README.md

```diff
@@ -39,6 +39,7 @@ Before you head over, read the [Contribution Guide](CONTRIBUTING.md) first. You
       - [Plotly](#plotly)
   - [Machine Learning](#machine-learning)
   - [Deep Learning](#deep-learning)
+  - [Reinforcement Learning](#reinforcement-learning)
```

Comment:
What do you think @massooti to include `Reinforcement Learning` in another section like `Machine Learning`?

Reply:
it's clear, academically both Deep Learning and Reinforcement Learning concepts, these are subfield of machine learning so you considered Deep Learning as a separated section, so i also considered rl as seprated section :cowboy_hat_face: